.github/workflows/build_and_test.yml

```diff
@@ -111,8 +111,18 @@ jobs:
       - name: Get postgres headers
         run: make postgres-headers -j$(nproc)
 
-      - name: Run cargo clippy
-        run: ./run_clippy.sh
+      - uses: taiki-e/install-action@cargo-hack
```

Comment:
It might be that this is unusable in our environment. Caching *should* however take care if this was just `cargo install cargo-hack`.

Reply:
We don't do any caching in this job.

Reply:
Will solve this by including cargo-hack in the rust image.